### ISSUE TITLE
Page content shifts around when switching light/dark theme due to different font sizes

### ISSUE THREAD

**Issue by jantari** [pglombardo/PasswordPusher#1787]
## 🐛 Bug Report

When switching between light and dark theme in by browsers devtools, using the default PasswordPusher theme, I noticed the page content was resizing / shifting up and down.

Looking into it I noticed it's because light and dark theme have different font-sizes configured:

light: https://github.com/pglombardo/PasswordPusher/blob/8d2be08e37c957635c48b7f1f689ace1ed413aa6/app/assets/stylesheets/themes/default.css#L61
dark: https://github.com/pglombardo/PasswordPusher/blob/8d2be08e37c957635c48b7f1f689ace1ed413aa6/app/assets/stylesheets/themes/default.css#L166

so basically switching to dark theme also, unexpectedly, made the site layout "more compact".

## 🔬 How To Reproduce

1. Run PasswordPusher with the default theme, no customizations
2. Open the passwordpusher page, e.g. at `http://localhost:5100`
3. Open your browsers devtools and repeatedly press the dark/light theme switcher button

### Code sample

no code required for this issue

### Environment

Where are you running/using Password Pusher?

- [ ] pwpush.com
- [X] Docker Image
  - [X] pwpush
  - [ ] custom image
- [ ] Heroku
- [ ] Digital Ocean
- [ ] Microsoft Azure
- [ ] Google Cloud
- [ ] AWS
- [ ] Source Code
- [ ] Other (please specify)

If applicable, what version of Password Pusher?

`release` tag, at the time of reporting this `1.36.5`.

## 📈 Expected behavior

There should be no font size differences between light and dark theme.

## 📎 Additional context

A workaround I tested to fix this is to add a `custom.css` with the following contents:

```css
@media (prefers-color-scheme: dark) {
  :root {
    --bs-body-font-size: 1.1rem;
  }
}
```

Theme switcher button location in Firefox DevTools:
![image](https://github.com/pglombardo/PasswordPusher/assets/25607767/a1c3b80e-2b24-40b7-9446-c1922697b222)

**Comment by jantari** [comment]
Because the fix for this is very simple I've opened a PR https://github.com/pglombardo/PasswordPusher/pull/1788

**Comment by pglombardo** [comment]
Excellent issue, fix & PR @jantari.  Much appreciated.  I'll merge and make a new release soon.

If you ever feel inclined, #1377 is another dark theme related bug.

**Comment by pglombardo** [comment]
This has been released in [v1.36.7](https://github.com/pglombardo/PasswordPusher/releases/tag/v1.36.7)
